src/client/java/teammates/client/scripts/sql/SeedDb.java

```diff
@@ -120,19 +122,53 @@ private void seedCourse() {
             }
 
             Random rand = new Random();
-
+            String courseId = UUID.randomUUID().toString();
             try {
                 String courseName = String.format("Course %s", i);
                 String courseInstitute = String.format("Institute %s", i);
                 String courseTimeZone = String.format("Time Zone %s", i);
-                Course course = new Course(UUID.randomUUID().toString(), courseName, courseTimeZone, courseInstitute,
+                Course course = new Course(courseId, courseName, courseTimeZone, courseInstitute,
                         getRandomInstant(),
                         rand.nextInt(3) > 1 ? null : getRandomInstant(), // set deletedAt randomly at 25% chance
                         false);
                 ofy().save().entities(course).now();
             } catch (Exception e) {
                 log(e.toString());
             }
+
+            seedFeedbackSession(courseId);
```

Comment:
Likewise, should this be in the try block? so that if there is an exception then you wont seed the feedback session

Reply:
In this case I think each seed function should not be catching the error within their own function but throwing to be caught here

Reply:
Thanks for catching this, i'll change in feedback question pr then.